The study presented here, uses constraint-based genome-scale metabolic modelling techniques in order to investigate metabolic commonalities and differences, and identify key targets in the adaptation process by integrating transcriptomics data. The relative fold change values of the gene expressions (deposited in GEO) of multiple resistant strains obtained by adaptive laboratory evolution are integrated into enzymatically constrained metabolic model of \emph{Saccharomyces cerevisiae}. Despite the low coverage of integrations caused by the limited genomic annotations on the Yeast8 model, the simulations that are carried out in a computational environment provided a clearer understanding on the metabolic changes for each strain compared to a sole differential expression analysis.

We observed that all the resistant mutants show different metabolic behaviors, despite the fact that two of the mutants, B2 and B8 strains, were adapted within the same environmental conditions. We also found recurring divergent points in multiple analyses of evolved strains. Glyceraldehyde-3-phosphate dehydrogenase, enolase, aldehyde dehydrogenase, pyruvate decarboxylase, enzymes of glutathione system reactions, coenzyme-A utilization, and many other reactions are found as the key points to investigate in search for an adaptive mechanism. Another notable finding from our study is that the metabolic models were able to capture similar behaviors in strains that have cross-resistance, and provided additional direction for further investigation to fatty acid metabolism. Our findings clearly emerged to a point that the enzymes that take part in the cell wall formation and the energy utilization pathways are the two most
