\chapter{MATERIALS AND METHODS}

\section{Consensus \emph{S. cerevisiae} Metabolic Model}
Variety of \emph{S. cerevisiae} genome-scale metabolic models have been used since 2003, and each reconstructed model introduced more manual curations, increasing gene numbers from annotations and better predictions regarding the previous ones \cite{lopes2017genome}. A consensus genome-scale metabolic model of \emph{S. cerevisiae}, Yeast8, is presented in an open-source, version-controlled maintainable way in 2019, claiming that the model can be represented and investigated in a systematic way using Git and GitHub as a hosting service for the model repository \cite{lu2019consensus}. Systematic way of Yeast8 enables to study simultaneously in collaborative studies, provides record keeping of model changes, version updates, where each version of it can be released periodically and accessible all the time (Figure \ref{fig:yeast8_github}).

Yeast8 model can be considered as an updated version of Yeast7 \cite{aung2013revising} with additional corrections based on the annotations available in KEGG and ChEBI, and several gene inclusions from the model iSce926 \cite{chowdhury2015using}. Final version of Yeast8 to date, version 8.3.4 released on July 28, has 3991 reactions, 2691 metabolites, 1149 genes and 14 intracellular compartments. Additional statistical analysis on its stoichiometric matrix can be seen in Figure \ref{fig:modelstats}.

All simulations in the Methods section are done on the model Yeast8 v8.3.4 which is hosted in Github. Optimization problems were solved with the COBRA Toolbox v3.0.6 and the RAVEN Toolbox v2.3.1 in MATLAB (version 9.7.0.1216025 (R2019b)), using Gurobi solver (version 8.1.1).

\begin{figure}[H]
\begin{center}
\includegraphics[width=0.8\columnwidth]{yeast8_github.png}
\end{center}
\caption[Repository structure of the model yeastGEM on GitHub \cite{lu2019consensus}]{Repository structure of the model yeastGEM on GitHub \cite{lu2019consensus}.}
\label{fig:yeast8_github}
\end{figure}

\begin{figure}[H]
\begin{center}
\includegraphics[width=1\columnwidth]{modelstats.png}
\end{center}
\caption[Coefficients and singular values of the stoichiometric matrix of Yeast8.]{Coefficients and singular values of the stoichiometric matrix of Yeast8.}
\label{fig:modelstats}
\end{figure}



\section{Experimental Data Acquisition} \label{experimentaldataacquisition}
Extracellular metabolomics data are obtained from Cakar's Lab \cite{arslan2018physiological}. Briefly, they performed ethyl methane sulfonate (EMS) mutagenesis on the prototrophic \emph{Saccharomyces cerevisiae} strain CEN.PK 113-7D (MATa, MAL2-8c, SUC2) as reference, and the cells were successively batch cultivated under gradually increasing levels of stress factors as a selection strategy by evolutionary engineering. After selection, the evolved strains were inoculated in 2\% Yeast Minimal Media (YMM), and the extracellular concentrations of glucose, ethanol, glycerol and acetate were measured at different time points by using HPLC. OD\textsubscript{600} values were determined by a spectrophotometer. Additionally, cell dry weight analysis was conducted to determine biomass production. Measured OD\textsubscript{600} values and dry weights of the reference strains (without mutagenesis) of all ALE experiments were used in this study, and are collected in Table \ref{table:experimental_OD600s_and_growths}. Experimental extracellular metabolite concentrations are also collected in Table \ref{table:experimental_data}, and compared with the computational simulation results in the discussions.
\vspace{0.5cm}

\input{tables/experimental_concentrations.tex}


As the slope in the curve of lnOD\textsubscript{600} as a function of time gives the growth rates of cells, natural logarithm of OD\textsubscript{600} values were calculated to obtain specific growth rates by using the Equation (\ref{eq:growthrates}),
  \begin{equation}
      \ \mu = \frac{\Delta \ln{OD_{600}}}{\Delta t}
      \label{eq:growthrates}
  \end{equation}
where $OD_{600}$ is the optical density of a sample measured at a wavelength of 600 nm, and \emph{t} is the time.

\vspace{0.5cm}
\input{tables/experimental_OD600s_and_growths.tex}

In order to determine uptake and secretion rates of the metabolites, the steady-state assumption is applied in three hours intervals as the shortest measured time-points. Missing data on cell dry weights are estimated from the OD\textsubscript{600} values, and these cell dry weight data is used to calculate fluxes (in the unit of mmol/(gDW)\textsuperscript{-1}). Measurement of the cell dry weight at the 3rd hour was crucial for the steady-state assumption, however data was not available from the experiments. Curve trend of the OD\textsubscript{600} plot is used as a guide to estimate cell dry weight (Figure \ref{fig:GrowthGraphs}).

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=0.9\columnwidth]{GrowthGraphsSmall.eps}
  \end{center}
  \caption[OD\textsubscript{600}, lnOD\textsubscript{600}, cell dry weights and growth rates]{OD\textsubscript{600}, lnOD\textsubscript{600}, cell dry weights and growth rates graphs. Estimated missing cell dry weight data is shown in red color.}
\label{fig:GrowthGraphs}
\end{figure}


\vspace{-0.5cm}

\section{Flux Balance Analysis}
Flux balance analysis (FBA) assumes that the living cells act as they optimized their lives towards some goal, and as if they were at steady state. To be more clear, steady-state assumption indicates that the metabolites are both produced and consumed at the same rate in a cell, without an accumulation. Therefore, in this system, metabolites are constrained by only the stoichiometric coefficients arise from mass balance of metabolites. FBA solves a set of ordinary differential equations regarding to the stoichiometric matrix:
\begin{equation}
 \ S_{m \times n} \cdot v=0
\end{equation}
\noindent where S is the matrix of the stoichiometric reaction coefficients with m number of metabolites (as rows) and n number of reactions (as columns), and v is the vector of all associated reaction fluxes (mmol/(gDW)\textsuperscript{-1}). Because the matrix S usually has more reactions than metabolites (m<n), the system can result multiple solutions, and being called an underdetermined system (read more on Section \ref{metabolicnetworks}). To solve it for an optimal solution, additional constraints or an objective function is required.

A "growth reaction" is usually included in the reactions of the system to represent the "goal" in the definition of living systems. Growth reactions act as the final consumption of metabolites necessary for the biomass production or cell replication. Additional to the growth, several exchange reactions (uptake or secretion of metabolites from or into extracellular space) are also included. Since the concentrations of extracellular metabolites are measurable experimentally, constraints can be applied to exchange reaction fluxes to shrink solution space. The more constraints introduced into the system, such as reversibility of reactions or known rate values, the more smaller solution space results. The growth reaction is usually used as an objective function to determine a unique solution from this solution space. The linear problem appears as,
\begin{align}
 \ \text{max}_v \quad & c^T \cdot v \\
 \label{eq:fba}
 \ \text{subject to} \quad & S_{m \times n} \cdot v=0 \\
 \ & v_{lb} \leq v \leq v_{ub}
\end{align}
\noindent where c is the objective function vector, v is the vector of fluxes, S is the stoichiometric matrix as in above equation. Subscripts lb and ub are the lower and upper boundaries on v. These constraints define a feasible region of the problem. The coefficients of the biomass constituents are defined as the same as the batch conditions in the reference article \cite{nilsson2016metabolic}, for the reason that detailed knowledge is not available in the acquired experimental data (see section \ref{experimentaldataacquisition}). The coefficients for the final biomass equation can be found in the Table \ref{table:biomass_coefficients}.

\vspace{0.5cm}

\input{tables/biomass_composition.tex}

There are multiple available command-line functions to solve a linear programming problem such as \emph{linprog()} function of MATLAB, \emph{optimizeCbModel()} function of the COBRA Toolbox, or \emph{solveLP()} function of the RAVEN Toolbox with using variety of solvers such as GLPK, GUROBI or IBM cplex.

In order to simulate batch conditions where the YMM is used as media, all the exchange reactions in the model are first blocked (lower bounds are set to 0). Then, only the exchange reactions of the ions that are available to the cells in the experimental design (ammonium, phosphate, sulphate, iron\textsuperscript{2+}, H\textsuperscript{+}, water, chloride, Mn\textsuperscript{2+}, Zn\textsuperscript{2+}, Mg\textsuperscript{2+}, sodium, Cu2\textsuperscript{2+}, Ca\textsuperscript{2+}, potassium) are set free (lower bounds are set to -1000), meaning that cells can uptake then as needed.


\section{Integration of Enzymatic Constraints}

  Solution space of the yeast model can be further constrained by limiting fluxes of reactions with integration of enzymatic information by using GECKO (GEM with Enzymatic Constraints using Kinetic and Omics data) method \cite{sanchez2017improving}.  In GECKO methodology, if a reaction has an annotated enzyme, enzyme entity itself is included in the same reaction equation. Stoichiometric coefficient of this entity is defined by its kinetic information and abundance in the cell (Figure \ref{fig:gecko1}). S matrix of the system is expanded with the addition of new rows for the enzymes and new columns for each enzyme's usage.

\begin{figure}[H]
\begin{center}
\includegraphics[width=1\columnwidth]{gecko1.png}
\end{center}
\caption[GECKO methodology to integrate enzymatic constraints\cite{sanchez2017improving}]{GECKO methodology to integrate enzymatic constraints\cite{sanchez2017improving}.}
\label{fig:gecko1}
\end{figure}

Metabolites A and B can be written in equations such as,
\begin{align}
\label{eq:geckoreaction}
 \ A \xrightarrow{E_i} B \\
 \label{eq:gecko}
 \ A + n_{ij}E_i \xrightarrow{} B
\end{align}
\noindent where $E_i$ is the enzyme catalyzing the convertion of A to B. The idea behind this methodology arises from the biochemical constraints, such that the maximum rate of a reaction is smaller or equal to its enzyme's turnover number and the concentration (mmol/gDW), and the equation,
\begin{align}
 \label{eq:gecko_simple}
 \ v_{j} \leq k_{cat}^{ij} \cdot [E_{i}]
\end{align}
\noindent shows the condition for the reaction flux depending on enzyme concentration. Enzymes in the reaction equations must be considered as pseudo-metabolites because they do not affect the mass balance of the reaction, i.e., enzymes are not consumed in the reactions but they are occupied for a short period of time. To maintain mass balances of enzymes, an overall enzyme usage pseudo-reaction is introduced into system in order to supplement enzymes into reactions, similar to exchange reactions, as the equation,
\begin{align}
 \label{eq:gecko_suppelement}
 \  EU_i: \quad \to E_i
 \
\end{align}
\noindent shows the draw reaction of an individual enzyme. If the flux carried by this reaction is $e_i$, we can say that $e_i$ ranges from 0 (i.e., no enzyme available) to a maximum value of $[E_i]$ (i.e., all the enzyme is used in the reaction),
\begin{align}
 \label{eq:gecko_enzymeflux}
 \  0 \leq e_i \leq [E_i]
\end{align}
then a mass balance for the enzyme $E_i$ can be defined under steady state assumption, the equation
\begin{align}
 \label{eq:gecko_enzymemass}
 \  -n_{ij} \cdot v_j + e_i = 0
\end{align}
arises. By rearranging the Equation (\ref{eq:gecko_enzymeflux}) and Equation (\ref{eq:gecko_enzymemass}), we obtain the equation,
\begin{align}
 \label{eq:gecko_enzymemassrearranged}
 \ v_j \leq \frac{1}{n_{ij}} \cdot [E_i]
\end{align}
and, if we compare this equation with the Equation (\ref{eq:gecko_simple}), we obtain,
\begin{align}
 \label{eq:gecko_enzymestoichiometry}
 \ n_{ij}=\frac{1}{k_{cat}^{ij}}
\end{align}
where we can finally calculate the stoichiometric coefficient for the enzyme. However, the Equation (\ref{eq:gecko_simple}) does not hold in all cases since reactions can be more complex. For isozymes in which multiple enzymes catalyze the same reaction, the equation becomes,
\begin{align}
 \label{eq:gecko_isozymes}
 \ v_{j} \leq \sum\limits_{i} k_{cat}^{ij} \cdot [E_{i}]
\end{align}
and, since the reaction can be catalyzed by all the isozymes equally, new reactions can be defined for each isozyme (note that each enzyme can have different $k_{cat}$ values). For example, if we think that the Equation (\ref{eq:geckoreaction}) had two isozymes, $A \xrightarrow{E_1 or E_2} B$, the new reactions,
\begin{align}
 \ R_{j/1}: \quad \frac{1}{k_{cat}^{1j}}E_1 + A \to B \\
 \ R_{j/2}: \quad \frac{1}{k_{cat}^{2j}}E_2 + A \to B
\end{align}
would be defined. These additional reactions distrupt the boundaries for the initial reaction. In order to maintain original boundaries, another "arm-reaction",
\begin{align}
 \ R_{j/arm}: \quad A \to M_{int} \\
 \ R_{j/1}: \quad \frac{1}{k_{cat}^{1j}}E_1 + M_{int} \to B \\
 \ R_{j/2}: \quad \frac{1}{k_{cat}^{2j}}E_2 + M_{int} \to B
\end{align}

\vspace{0.5cm}

\noindent is also introduced to the system where it acts as an intermediate metabolite. For an enzyme complex catalyzing a single reaction, since the reaction is catalyzed by all the enzymes which share the same $k_{cat}$ value, the Equation (\ref{eq:gecko_simple}) becomes,
\begin{align}
 \label{eq:gecko_complex}
 \ v_{j} \leq  k_{cat}^{ij} \cdot \min\limits_{k} \frac{ [U_{ik}] }{ S_{ik} }
 \
\end{align}
\noindent where $[U_{ik}]$  is the concentration of the subunit of the catalyzing enzyme $E_{i}$, and $S_{ik}$ is the stoichiometry of the subunit. For the reaction, if we think that the Equation (\ref{eq:geckoreaction}) was catalyzed by an enzyme complex of two subunits, namely ${E_1}$ and ${E_2}$, the equation,
\begin{align}
 \label{eq:gecko_complex2}
 \ R_{j}: \quad \frac{S_1}{k_{cat}^{ij}}E_1 +\frac{S_2}{k_{cat}^{ij}}E_2 + A \to B
\end{align}
where two of the subunits for the enzyme complex included in.

Finally, for the reversible reactions, two reactions must be defined for both forward and backward direction reactions with the same catalyzing enzyme but possibly with different $k_{cat}$ values. Assume the Equation (\ref{eq:geckoreaction}) is reversible, the equations,
\begin{align}
 \label{eq:gecko_reversible}
 \ R_{j/f}: \quad \frac{1}{k_{cat}^{ij/f}}E_i + A \to B \\
 \ R_{j/b}: \quad \frac{1}{k_{cat}^{ij/b}}E_i + B \to A
\end{align}
would be introduced to the system.

\section{Differential Expression Analysis and Integration}
Normalized gene expression profiles of the GSE series (Table \ref{table:gse_experiment}) obtained from the Gene Expression Omnibus are first differentially analyzed using Biobase \cite{huber2015orchestrating}, GEOquery \cite{davis2007geoquery} and limma \cite{ritchie2015limma} packages in R environment. Briefly, design matrices are constructed for each ALE experiment separately. Contrast matrices are obtained to make pair-wise comparison between the reference and the evolved strains. Linear models for each gene are fitted to the expression data for each contrast matrix, and empirical Bayes method was used on the fitted models for the assessment of differential expressions. Genes with the p-values of $<$0.05 are selected to be integrated into the constrained-based model.

\vspace{-0.5cm}

\input{tables/gse_experiment.tex}

\vspace{-1cm}

Obtained relative fold changes of the genes are integrated into metabolic model as the multipliers of the enzyme coefficients that are used to draw enzymes from the enzyme pool as in Equation (\ref{eq:gecko_suppelement}). In that sense, the gene products, i.e., the proteins with higher fold-change values ($>1$) are considered over-expressed and they are easily drawn from the pool, while proteins with lower fold change values ($<1$) are considered under-expressed and the reactions that are using these enzymes are become more costly to the system.



\section{Phenotype Phase Plane and Robustness Analysis}

As previously mentioned, there is no single solution to the linear problem of the metabolic model. Phenotype phase planes (PhPP) are used to describe all the feasible metabolic states in two or three dimensional surfaces, depending on the number of metabolites chosen to see how they affect the objective function \cite{edwards2002characterizing}. Commonly for aerobic models, various levels of glucose and oxygen availability through their uptake reactions are used to generate PhPP surfaces in three dimensions with the objective function of growth. Phenotype phase plane construction refers to a double robustness analysis on the model for selected reactions. Fundamentally, a phenotype phase plane is a two-dimensional plane defined by axes of the exchange rates of the selected nutrients. Since there are finite number of points in this plane, flux balance analysis finds the optimal flux distribution with the maximum or the minimum value of the objective function from all the existing points. The robustness of the network is determined by calculating the optimal flux distribution over iteratively altering the selected nutrient exchange reaction boundaries from maximum (100\%) to minimum (0\%). Briefly, robustness analysis shows how the objective changes as the flux through the selected reaction varies in magnitude \cite{edwards2002characterizing}.

In Figure \ref{fig:phpp_example}, an example of a phenotypic phase plane for the specific growth rate as a function of two substrates (oxygen and glucose uptake rates) is shown. Four sections in the figures represent the four phases in the cell, i.e., four different metabolic states. In the first section (at low glucose uptake rates), after the investigation on the flux vector, it can be concluded that glucose is fully consumed and converted into carbon dioxide through TCA cycle. In the following sections, several byproducts in the form of weak acids are secreted (acetic acid in the second phase; acetic and formic acid in the third phase). In the fourth phase, ethanol is secreted additional to the acetic and formic acid. Line of optimality (LO) in the phenotype phase planes shows the optimal boundary where the carbon source (glucose) is converted into biomass at a maximum yield.

\begin{figure}[H]
\begin{center}
\includegraphics[width=1\columnwidth]{figures/phpp_example.png}
\caption[An example of a phenotypic phase plane for the specific growth rate as a function of oxygen and glucose uptake rates as (A) 3D plot and (B) contour plot]{An example of a phenotypic phase plane for the specific growth rate as a function of oxygen and glucose uptake rates as (A) 3D plot and (B) contour plot.}
\label{fig:phpp_example}
\end{center}
\end{figure}


\section{Flux Variability Analysis}
Flux variability analysis (FVA) finds the minimum and maximum available fluxes for each reaction while obeying the provided constraints (for example fixed glucose uptake or growth rate). FVA is mainly used  1) to evaluate the robustness of the model \cite{thiele2010functional}, 2) to find alternative optimum states \cite{mahadevan2003effects}, 3) to check flux distributions when growth is not at optimum level \cite{reed2004genome}. Many other applications of FVA can be found in the literature \cite{gudmundsson2010computationally}.

Flux variability analysis, similar to FBA, solves two optimization problems for each reaction,
\begin{align}
\ \text{max}_v / \text{min}_v \quad & v_i \\
\ \text{subject to} \quad & S_{m \times n} \cdot v=0 \\
\ & w^T \cdot v \geq \gamma \cdot Z_0 \\
\ & v_{lb} \leq v \leq v_{ub}
\end{align}
\noindent where $w$ is the objective function equals to $c$ in the problem, Equation (\ref{eq:fba}), $Z_0 = w^T \cdot v_0$ describes an optimal solution to the problem, Equation (\ref{eq:fba}), $\gamma$ is an indicator to check whether the FVA is done at the optimal state (where objective flux is the same and $\gamma = 1$) or any other state (where $0 \leq \gamma < 1$).


\section{Random Sampling of Solution Space}
Constraints applied to a model define a solution space, a convex polytope, where every flux distribution is accessible. Random sampling of the solution space is an unbiased tool to explore metabolic models. Mainly, Markov Chain Monte Carlo methods are used to sample this space using algorithms such as (Artificially Centered) Hit-and-Run (HRB) \cite{kiatsupaibul2011analysis, saa2016ll} algorithm, and this method has proven to be helpful in the analysis of genome-scale metabolic models \cite{schellenberger2009use}. The COBRA Toolbox \cite{heirendt2019creation} provides a workflow to generate random points by using the Hit-and-Run algorithm. Briefly, the random sampling method collects points that are uniformly distributed in the solution space and calculates the most probable flux value for each reaction.

Since the computational burden of the loopless sampling is high, generated random points in the solution space of GECKO integrated Yeast8 model include thermodynamically unfeasible states. To this sense, instead of using Hit-and-Run algorithm to sample inside the region of allowed solutions, corners of the solution space is sampled by using the workflow provided by the RAVEN Toolbox \cite{wang2018raven}.

The workflow from the RAVEN Toolbox uses the simplex method, one of the main algorithms in mathematical optimization, to calculate corners of the solution space with a random set of objective functions. Maximization of these random functions (reactions in our case) brings out the corners of the allowed solution space. This workflow also includes flux variability analysis from the COBRA Toolbox in order to prevent internal loops.

\section{Minimization of Metabolic Adjustment}
An approach for predicting mutant behavior similar to FBA is based on the Minimization of Metabolic Adjustment (MOMA). The idea behind this methodology arises because a mutant is more likely to display a sub-optimal flux distribution since the selection through evolutionary adaptation has altered the genetic background of the evolved strains. The application of FBA on both wild-type and mutant strains is considered insubstantial. While the FBA method uses the objective function to present an optimal solution, MOMA choses the objective function as the minimization of the Euclidean distance between the optimal flux distribution calculated by FBA on the reference strain and the sub-optimal flux distribution of the mutant where it remains initially as close as possible to the optimal flux distribution of the wild-type.

Distance in the flux space is a quadratic function, thus, quadratic programming (QP) algorithm is used in MOMA instead of linear programming (LP). The COBRA Toolbox provides an approach in the form of a function called \emph{MOMA()} which can easily be implemented with command line in MATLAB. Within the function, the following FBA problem is solved first,
\begin{align}
 \ \text{max} \quad & c^T_{wt} \cdot v_{wt} \\
 \label{eq:moma_1}
 \ \text{subject to} \quad & S_{wt} \cdot v_{wt}=0 \\
 \ & lb_{wt} \leq v_{wt} \leq ub_{wt}
\end{align}
and then the second problem is solved,
\begin{align}
 \ \text{min} \quad & \sum (v_{wt} - v_{mut})^2\\
 \label{eq:moma_2}
 \ \text{subject to} \quad & S_{wt} \cdot v_{wt}=0 \\
 \ & S_{mut} \cdot v_{mut}=0 \\
 \ & lb_{wt} \leq v_{wt} \leq ub_{wt} \\
 \ & lb_{mut} \leq v_{mut} \leq ub_{mut} \\
 \ & c^T_{wt} \cdot v_{wt} = f_{wt}
\end{align}
\noindent
where $f_{wt}$ is the optimal objective value found by FBA in the first problem.
