\chapter{RESULTS}

\section{Model Constraints and Curation}


To make sure the \emph{in-silico} growth rate predictions are in agreement with the physiological kinetic parameters obtained from the laboratory experiments, fine adjustment on the chosen metabolic model is a requirement. Since the growth-associated maintenance (GAM) and non-growth associated maintenance (NGAM) reactions in a model play a determinant role in the simulation results, fluxes through these reactions must be constrained to a fixed value. Flux of the NGAM reaction is constrained to 0.7 mmol/gDWh\textsuperscript{-1} for aerobic model, and 0 mmol/gDWh\textsuperscript{-1} for anaerobic model simulations as calculated in the previous studies \cite{nilsson2016metabolic}. Since the GAM depends on the biomass composition defined, results of a chemostat experiment \cite{van1998effect} are used as a guide to fit simulation predictions to. The model is simulated iteratively with a range of values for the GAM and the best fit is found at the level of 31.4 mmol/gDWh\textsuperscript{-1} (Figure \ref{fig:gam_fitting}).

\begin{figure}[H]
  \begin{center}
      \includegraphics[width=1\columnwidth]{figures/gamfitting.png}
      \caption[The required flux for the growth associated maintenance reaction is determined by the fitting simulation results to the experimental data]{The required flux for the growth associated maintenance reaction is determined by the fitting simulation results to the experimental data. The best fit is found at 31.4 mmol/gDWh\textsuperscript{-1} and the reaction is constrained to that flux value.}
      \label{fig:gam_fitting}
  \end{center}
\end{figure}

Average enzyme saturation value is set to 50\% where the lowest error rate is obtained as a fraction to biomass (Figure \ref{fig:sigma_fitting}). By constraining the total enzyme mass, the metabolic model became able to show overflow metabolism without any other constaint applied.

\begin{figure}[H]
  \begin{center}
    \includegraphics[width=1\columnwidth]{figures/sigmafitting.png}
    \caption[Average enzyme saturation factor for the GECKO method is determined by the iterative simulations for the lowest absolute relative error to the experimantal data]{Average enzyme saturation factor for the GECKO method is determined by the iterative simulations for the lowest absolute relative error to the experimantal data. The value is set to 50\% where the lowest error rate is obtained.}
    \label{fig:sigma_fitting}
  \end{center}
\end{figure}

To prevent over-constraining the model with the enzyme kinetics, the most limiting proteins in the model are found based on their sensitivity on the objective function, i.e. growth. The $k_{cat}$ values for the the limiting enzymes are updated with the maximum available values from BRENDA\cite{jeske2019brenda} database by automated iterations until the simulation results for the growth rate agrees with the experimental growth rate. These alterations for enzymes with their objective control coefficients can be found in the Table \ref{table:gecko_iterations}.

\input{tables/gecko_iterations.tex}

\vspace{-0.5cm}
\section{Differential Expression Analysis and Integration}

The obtained expression datasets were quantile-normalized for their own experiments. Three replicates for both reference and the evolved strains for each experiment (exception with the ethanol-b8 strain which has two replicates) is used for the differential expression analysis. The distribution across normalized gene expression levels across each experiment can be seen in the Figure \ref{fig:expr_boxplot}. No further normalization applied to prevent introducing errors and datasets are used as obtained from the database.

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/expr_boxplots.png}
  \caption[Boxplots of the normalized gene expression levels]{Boxplots of the normalized gene expression levels obtained from Gene Expression Omnibus. Quantile normalization performance is indicated by the medians (black horizontal lines) that are almost at the same level with their reference reads.}
  \label{fig:expr_boxplot}
  \end{center}
\end{figure}

A statistical threshold (p $<$ 0.05) for the differential analysis of the normalized gene expression levels detected a total of 1606 genes in ethanol-b2, 2947 genes in ethanol-b8, 4743 genes in caffeine, 2267 genes in coniferylaldehyde, 3448 genes in iron, 152 genes in nickel, 4796 genes in phenylethanol, and 4796 genes in silver resistant strains differentially expressed. Commonly upregulated and downregulated genes frequencies across all strains can be found in the Figure \ref{fig:intersections_by_degree}.



\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/intersections_by_degree.png}
  \caption[UpSet plot showing the number of common genes that are differentially expressed significantly]{UpSet plot showing the number of common genes that are differentially expressed significantly (p $<$ 0.05) across evolved strains. Connected black circles in the below panel's matrix represents a certain intersection between experiment strains, and the number of genes in that intersection are shown in the top bar graphs. Differentially expressed genes are plotted seperately for (A) upregulated genes, and (B) downregulated genes.}
  \label{fig:intersections_by_degree}
  \end{center}
\end{figure}

Despite the high number of differentially expressed genes in the silver resistant strain, the intersection with the most number of strains discluded it. There were not any common differentially expresssed genes considering all strains. The most common genes (as shown in the first bars in the UpSet plot) are reported in Table \ref{table:common_genes}.

\begin{table}[H]
\small
\vskip\baselineskip
  \begin{center}
  \caption[Common upregulated and downregulated genes]{The common upregulated and downregulated genes within ethanolb2, ethanolb8, caffeine, coniferylaldehyde, iron, nickel and phenylethanol resistant strains.}
    \vspace{5mm}
  \begin{tabular}{|p{4cm}|p{10cm}|}
     \hline
    \textbf{Upregulated Genes} & \textbf{Downregulated Genes}  \\
      \hline
      YDR075W, YDR492W, YGR239C, YIL118W, YLR215C, YML005W, YML125C, YNL024C, YNL188W, YNR044W, YOL002C, YOR107W, YPR052C & YBL015W, YBR132C, YBR285W, YBR299W, YCR068W, YDL238C, YDR055W, YER033C, YGL146C, YGL250W, YGR023W, YGR149W, YGR161C, YGR288W, YGR289C, YGR292W, YIL097W, YIR017C, YJL042W, YJL053W, YJL082W, YJL132W, YLR092W, YLR176C, YLR240W, YLR446W, YMR052C-A, YMR053C, YMR081C, YMR103C, YMR104C, YMR105C, YMR160W, YMR194C-A, YMR258C, YMR291W, YNL093W, YNL277W, YNR001C, YOL117W, YOR027W, YOR132W, YOR178C, YOR230W, YPR079W, YPR154W \\ \hline
  \end{tabular}

  \label{table:common_genes}
  \end{center}
\end{table}



The log2(fold-change) values of differentially expressed genes and their frequencies are plotted for each experiment in Figure \ref{fig:expr_frequencies_before}. Although the high number of differentially expressed genes are collected, due to the limited reaction number in the genome-scale metabolic models, only a part of obtained fold-change expression values were able to be integrated. Because of the limitations in the model, no further cutoff considered for the fold change values.

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/expr_frequencies_before.png}
  \caption[Frequencies of the differentially expressed genes after the gene expression analysis]{Frequencies of the differentially expressed genes after the gene expression analysis.}
  \label{fig:expr_frequencies_before}
  \end{center}
\end{figure}

Total of 277 genes in ethanol-b2, 464 genes in ethanol-b8, 783 genes in caffeine, 374 genes in coniferylaldehyde, 552 genes in iron, 28 genes in nickel, 759 genes in phenylethanol and 759 genes in silver resistant strains were integrated into the metabolic model to generate strain-specific models (Figure \ref{fig:expr_frequencies_after}).

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/expr_frequencies_after.png}
  \caption[Frequencies of the differentially expressed genes that are integrated into the metabolic model]{Frequencies of the differentially expressed genes that are integrated into the metabolic model.}
  \label{fig:expr_frequencies_after}
  \end{center}
\end{figure}

In comparison, 14.13\% of genes in ethanol-b2, 15.74\% of genes in ethanol-b8, 16.51\% of genes in caffeine	16.50\% of genes in coniferylaldehyde, 16.01\% of genes in iron, 18.42\% of genes	in nickel, 15.83\% of genes in phenylethanol and 15.83\%  of genes in silver resistant strains were found in the metabolic model and they are integrated.

In order to validate the expression data integration method, fluxes for each enzyme (i.e. protein flux from protein pool to reaction for each protein) is predicted by flux balance analysis on all the strain models and on the wild-type model (wild-type here is referred as the metabolic model without expression data integrated). Flux ratios between wild-type and evolved models are calculated, and linear regression analysis is fitted after elimination of the outliers by the generalized extreme studentized deviate test to compare the flux-changes with the protein fold-changes from the expression data. Results confirm a good correlation between simulations and protein fold-change values (Figure \ref{fig:expr_linear_regressions}).

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=0.8\columnwidth]{figures/expr_linear_regressions.png}
  \caption[Linear regression analyses of the fold-changes]{Linear regression analyses of the differential expression fold-changes obtained from microarray studies and the enzyme-draw flux fold-changes obtained from flux balance analysis simulations of corresponding proteins.}
  \label{fig:expr_linear_regressions}
  \end{center}
\end{figure}



\section{Flux Balance Analysis Simulations}

The metabolic model is first simulated without the integration of the expression data (this model will be referred as wild-type model). Since the model is already enzymatically constrained, the only additional constraint required is the total amout of enzymes available in the enzyme pool. All the lower bounds for reactions were set to 0, and the upper bounds were set to 1000, except for the enzyme pool reaction. By doing that, the model was set free to be able to allocate enzymes as the reactions require. Flux rates of several exchange reactions as a function of growth rate are plotted in Figure \ref{fig:wt_crabtree}.

A behavioral change is observed around the time when growth rate is approximately 0.3 h\textsuperscript{-1}. Since the model is simulated under fully areobic conditions (no constraints applied) and produces ethanol, this behavior can be explained by the Crabtree effect, where the yeast metabolism switches to perform both respiration and fermentation at the same time at a critical specific growth rate.

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/wt_crabtree.png}
  \caption[Metabolic model shows the overflow metabolism]{Metabolic model shows the overflow metabolism (crabtree effect) under fully aerobic conditions.}
  \label{fig:wt_crabtree}
  \end{center}
\end{figure}

Robustness analysis is performed on the wild-type model for the growth rate with the varying levels of glucose uptake, oxygen uptake, acetate secretion and ethanol secretion rates (Figure \ref{fig:wt_robustness}). Unlike the traditional metabolic models, enzymatically constrained model showed non-linear graphs.

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/wt_robustness.png}
  \caption[Robustness analyses on the glucose uptake, oxygen uptake, acetate secretion and ethanol secretion rates]{Robustness analyses on the glucose uptake, oxygen uptake, acetate secretion and ethanol secretion rates.}
  \label{fig:wt_robustness}
  \end{center}
\end{figure}

After the validation of the wild-type model, all models for strains are simulated for the growth rates as a function of iteratively increasing glucose uptake rates (Figure \ref{fig:growth_glucose_ales}). Models were able to grow up to the point where the protein accesibility becomes a limiting factor. It has been found that the evolution models except for the caffeine tolerant model show similar patterns on the points where the enzymes become limiting (break points in the lines). Caffeine tolerant model shows different pattern in a way that it can consume more glucose to reach higher growth rates compared to others. Also it must be noted that while only caffeine and coniferylaldehyde tolerant models can grow at higher rates than wild-type model, ethanol, iron, nickel, phenylethanol and silver models can grow at lower maximum rates. The maximum growth rates and corresponding glucose uptake rates for each model are summarized in the Table \ref{table:growth_glucose_table}.

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/growth_glucose_ales.png}
  \caption[Growth rates as a function of iteratively increasing glucose uptake rates]{Growth rates as a function of iteratively increasing glucose uptake rates.}
  \label{fig:growth_glucose_ales}
  \end{center}
  \end{figure}
\vspace{-1.0cm}



\begin{table}[H]
\small
\vskip\baselineskip
  \begin{center}
  \caption[Maximum growth rates obtained from flux balance analysis simulations for each strain and their maximum glucose uptake rates (GUR) required for the growth]{Maximum growth rates obtained from flux balance analysis simulations for each strain and their maximum glucose uptake rates (GUR) required for the growth.}
    \vspace{5mm}
  \begin{tabular}{|c|c|c|}
     \hline
    \textbf{Experiment} & \textbf{Growth Rate (1/h)} & \textbf{GUR (mmol/gDWh)} \\
      \hline
      ethanolb2           & 0.35466                    & 14                                       \\ \hline
      ethanolb8           & 0.34736                    & 16                                       \\ \hline
      caffeine            & 0.46769                    & 28                                       \\ \hline
      coniferylaldehyde   & 0.39132                    & 20                                       \\ \hline
      iron                & 0.37554                    & 15                                       \\ \hline
      nickel              & 0.37523                    & 18                                       \\ \hline
      phenylethanol       & 0.34382                    & 18                                       \\ \hline
      silver              & 0.34163                    & 25                                       \\ \hline
      wildtype            & 0.37618                    & 19                                       \\ \hline
  \end{tabular}
  \label{table:growth_glucose_table}
  \end{center}
\end{table}

Phenotype phase planes are obtained through double robustness analysis to see the effect of oxygen on growth while different glucose levels are fed to the system. Contour plots showing different regions can be seen in the Figure \ref{fig:robustness_glu_oxy}.

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=0.85\columnwidth]{figures/robustness_glu_oxy.png}
  \caption[The glucose uptake rate (mmol/gDWh) versus oxygen uptake rate (mmol/gDWh) phenotype phase planes show the cellular growth rate as colored contours for each adapted model simulations]{The glucose uptake rate (mmol/gDWh) versus oxygen uptake rate (mmol/gDWh) phenotype phase planes show the cellular growth rate as colored contours for each adapted model simulations.}
  \label{fig:robustness_glu_oxy}
  \end{center}
  \end{figure}
\vspace{-1.0cm}

In order to characterize the differentiating reactions in the flux balance analysis solution vectors, cumulative flux vectors of each experiment are plotted (Figure \ref{fig:cumflux_free}). It must be remembered that at this point, the only constraint applied to the models is the level of enzymes available in the enzyme pool. In other words, the models are able to allocate enzymes as the reactions require while all other sources (carbon, nitrogen, etc.) are unlimited. Standart deviations for each reaction between strains are calculated to find most diverging points and plotted as a bar graph.

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/cumflux_free.png}
  \caption[Cumulative flux vectors of each model simulations under unlimited glucose uptake constraints and the standart deviations of reactions across each adaptation]{Cumulative flux vectors of each model simulations under unlimited glucose uptake constraints and the standart deviations of reactions across each adaptation.}
  \label{fig:cumflux_free}
  \end{center}
\end{figure}

Reactions with the standart deviation values more than 10 mmol/gDWh are listed in Table \ref{table:cumulative_free_stds}. The same reactions with different numbers in their names use different enzymes. For example, glyceraldehyde-3-phosphate dehydrogenase (GADPH) reaction can be carried out in the presence of YGR192C or YJL052W or YJR009C genes' proteins, in other words, TDH1 or TDH2 or TDH3 isozymes. Therefore there are No1, No2 and No3 versions of the same reaction, using different enzymes. It must be noted that these multiple reactions carried out with isozymes in the metabolic model may not mean a significant difference if the enzymes do not catalyze any other reaction. This means that the chosen isozyme may not matter mathematically if the flux value across the isozyme reactions are the same.
\input{tables/cumulative_free_stds.tex}

Because of the reason behind the differentiation is mostly caused by different uptake values of glucose, cumulative flux vectors of each experiment are plotted once more when the boundaries of the glucose uptake reaction is constrained to 10 gDWh\textsuperscript{-1} for each model (Figure \ref{fig:cumflux_gur10}). This has done to eliminate the differences caused by the free glucose uptake rates (as shown in below section, each model can uptake different maximum amount of glucose) and therefore affecting the downstream reaction fluxes.

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/cumflux_gur10.png}
  \caption[Cumulative flux vectors of each experiment when glucose uptake rate is constrained]{Cumulative flux vectors of each simulation and standart deviations. Glucose uptake rate is constrainted to 10 mmol/gDWh for all models. }
  \label{fig:cumflux_gur10}
  \end{center}
\end{figure}

Reactions with the standart deviation values more than 5 mmol/gDWh are listed in Table \ref{table:cumulative_free_stds}. Despite the decrease in the flux values (caused by the lower uptake rates of glucose), the top divergent reactions obtained are in agreement with the no-constraint flux balance analysis results.

\input{tables/cumulative_gur10_stds.tex}

Finally, only the arm reactions (where the isozyme reactions were seperated from, see Methods) when the glucose uptake rate is constrained to 10 mmol/gDWh are plotted as heatmap in Figure \ref{fig:fba_heatmap}. Arm reactions in this context refer to the combined reactions that normally have multiple sub-reactions because of isozymes catalyzing the same reaction. As it can be seen from the previous tables, most of the differences were caused by the isozymes with the flux values close to each other. By considering only arm reactions and a fixed glucose uptake rate, the reactions that actually differ because of the expression integration are found.

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/fba_heatmap.png}
  \caption[Heatmap of the reaction fluxes where the standart deviation between models are the highest (top 50). Reversible means the intake of a substrate, arm means the flux of the reaction consists of isozyme reactions]{Heatmap of the reaction fluxes where the standart deviation between models are the highest (top 50). Reversible means the intake of a substrate, arm means the flux of the reaction consists of isozyme reactions.}
  \label{fig:fba_heatmap}
  \end{center}
\end{figure}

Overall the most divergent strain can be seen as silver resistant strain while the nickel strain has the closest values to the wildtype model as expected considering the integrated expression data. The highest deviation caused in the water diffusion from mitochondria to cytoplasm indicates that even at the same glucose feeding rates, phenylethanol resistant strain requires higher (89.5\%) water in the mitochondria. Followed with the ferrocytochrome-c:oxygen oxidoreductase, pyruvate decarboxylase reactions and different exchange reactions, the results conclude that the models of evolved strains follow different paths to produce biomass with the same environmental conditions provided.

As these different models belong to evolved strains, hierarchical relationship between models are drawn as dendrogram in Figure \ref{fig:fba_dendrogram}. Euclidean pairwise distance is calculated in MATLAB between flux balance analysis solution vectors for each model where the glucose uptake rate is constrained to 10 mmol/gDWh first and then clustered by the unweighted average distance (UPGMA) method to draw a tree. Cophenetic correlation coefficient is found as 0.9744 while theSpearman's rank correlation between the dissimilarities and the cophenetic distances is calculated as 0.9096, indicating a high-quality solution.

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/fba_dendrogram.png}
  \caption[Hierarchical relationships between the strains drawn as a dendrogram plot. Euclidian distances are calcualted from the reaction fluxes (flux balance analysis) where the glucose uptake rate is constrained to 10 mmol/gDWh]{Hierarchical relationships between the strains drawn as a dendrogram plot. Euclidian distances are calcualted from the reaction fluxes (flux balance analysis) where the glucose uptake rate is constrained to 10 mmol/gDWh.}
  \label{fig:fba_dendrogram}
  \end{center}
\end{figure}

Despite the high correlation between the fold changes obtained from differential expression analysis and the flux balance analyses for particular enzymes, outliers must be investigated for the indirect regulation of enzymes. Three groups can be formed for these: Firstly, enzymes may show transcriptional regulation behaviors (TR) if their flux fold changes (between evolved and wildtype strain models) correlate with the differential gene expression results. Secondly, enzymes may show metabolic regulation behaviors (MR) if the flux change is observed even in the absences of the expressional changes. And lastly, enzymes may show post-transcriptional regulation behaviors (PR) if their expression levels change without showing any flux changes (Table \ref{table:regulation_table}). In the Figure \ref{fig:fba_regulation}, enzymes that have significant expressional fold changes (p $<$ 0.05) in all models are categorized for this purpose.

\vskip 1\baselineskip
\begin{table}[H]
\begin{center}
\caption[Enzymatic regulation decision table of expressional fold change (from differential analysis) and flux fold change (from flux balance analysis) comparison.]{Enzymatic regulation decision table of expressional fold change (from differential analysis) and flux fold change (from flux balance analysis) comparison.}
\vskip 0.5\baselineskip
\label{table:regulation_table}
\begin{tabular}{l|l|l|l|}
\cline{2-4}
\textbf{ExprFC \textbackslash FluxFC} & \textbf{Positive} & \textbf{Negative} & \textbf{No Change} \\ \hline
\multicolumn{1}{|l|}{\textbf{Positive}}  & Transcriptional & Metabolic       & Post-transcriptional \\ \hline
\multicolumn{1}{|l|}{\textbf{Negative}}  & Metabolic       & Transcriptional & Post-transcriptional \\ \hline
\multicolumn{1}{|l|}{\textbf{No Change}} & Metabolic       & Metabolic       &                      \\ \hline
\end{tabular}
\end{center}
\end{table}

Interestingly, most of the enzymes show post-transcriptional regulation behaviors indicating that even though the expression values are found significantly changed in the microarray analysis, their carried flux values did not differ in the \emph{in silico} simulations. Single simulation differences (such as FAS1, DFR1, CHO1 and ERG11 genes showing transcriptional regulation in all models except nickel model where they show metabolic regulation) must be investigated in order to understand the causal relationship.

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/fba_regulation2.png}
  \caption[Categorization of enzymes for regulation points for transcriptional regulation (TR), metabolic regulation (MR), post-transcriptional regulation (PR)]{Categorization of enzymes for regulation points for transcriptional regulation (TR), metabolic regulation (MR), post-transcriptional regulation (PR).}
  \label{fig:fba_regulation}
  \end{center}
\end{figure}

Shadow prices are calculated by solving the optimization problem for the objective function (growth rate) by increasing the upper constraints by one unit for all reactions. This way, the sensitivity of the reactions to the growth rate is calculated for all models. In Table \ref{table:fba_shadowprices_free_table}, the shadow prices of the reactions that limit the growth rate more than itself and their values can be seen. The values for each reaction are divided to the wild-type model's shadow price values to uncover the adaptive evolution effects and plotted in Figure \ref{fig:fba_shadowprices_free}.

\vskip 1\baselineskip
\input{tables/fba_shadowprices.tex}

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/fba_shadowprices_free.png}
  \caption[Barplots of the changes in the shadow prices of the metabolites that have contibutions higher than the biomass (growth) itself. Shadow price of each reaction for evolved models is divided by the shadow price of that reaction in the wildtype model]{Barplots of the changes in the shadow prices of the metabolites that have contibutions higher than the biomass (growth) itself. Shadow price of each reaction for evolved models is divided by the shadow price of that reaction in the wildtype model.}
  \label{fig:fba_shadowprices_free}
  \end{center}
\end{figure}

Shadow prices of metabolites show us that the metabolites zymosterol, ergosta-5,7,22,24(28)-tetraen-3beta-ol, ergosterol, lanosterol, 14-demethyllanosterol, biotin, riboflavin, flavin mononucleotide (FMN), 5-formyltetrahydrofolic acid, 4-aminobenzoate and hexanoate are the most required metabolites in the media to increase growth rate in all models, in order for their mean values across models. Although all the models would increase their biomass production with these metabolites supplied, increment amounts would differ for each model. While the silver resistant model would increase its growth rate by 250\% with biotin supplement, phenylethanol resistant model could only achieve 33\% increase.

When we look at the main ATP producing reactions in the system, wee see the reactions pyruvate kinase and phosphoglycerate kinase are actively used.


ADP + H+ + phosphoenolpyruvate \xrightarrow{pk} ATP + pyruvate

1,3-bisphospho-D-glycerate + ADP \xrightarrow{PGK1} 3-phosphonato-D-glycerate(3-) + ATP


Comparison between these ATP producing reactions under


\vskip 1\baselineskip
\input{tables/fba_atp_production.tex}

\section{Flux Variability Analysis}
Flux ranges of each enzyme to find used enzymes and pathways on models are analyzed by the linear problem in the Flux Variability Analysis method where the objective functions to minimize and maximize all reactions iteratively with growth rate as the objective function kept at 90\% of its maximum value. Minimum and maximum available fluxes are collected in the iterative process for each reaction, and results are plotted in Figure \ref{fig:fva_enzymes}). Standart deviations on flux ranges for each reaction are also plotted to find most divergent targets. .

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/fva_enzymes.png}
  \caption[Flux variability analysis results as flux ranges per enzyme usage reaction, sorted by the wild-type flux ranges]{Flux variability analysis results as flux ranges per enzyme usage reaction, sorted by the wild-type flux ranges.}
  \end{center}
  \label{fig:fva_enzymes}
  \end{figure}

As it can be seen in the Table \ref{table:fva_results} where the most divergent proteins (std $>$ 50) across all experiments are listed, paralog components of the glutathione system namely glutaredoxin-1 (GRX1) and glutaredoxin-2 (GRX2) shows the highest divergence across strains, followed by the glyceraldehyde-3-phosphate dehydrogenase (GAPDH) isozymes, triose-phosphate dehydrogenase 2 and 3 (TDH3 and TDH2).

\input{tables/fva_results.tex}

Pathway-wise investigation is carried out from the flux variability analysis results. First, the reactions that have subsystem annotations are collected for each seperately. It must be noted that some reactions can exist in multiple subsystems, for example, phosphoenolpyruvate carboxykinase reaction exists in all gluconeogenesis, glycolysis, citrate cycle (TCA cycle), pyruvate metabolism, biosynthesis of secondary metabolites, biosynthesis of antibiotics, and carbon metabolism subsystems. Secondly, the mean flux values for each subsystem for all models in flux variability analysis are collected. These flux values for the most common subsystems are plotted as stacked bars in Figure \ref{fig:fva_subsystem_spans} to distinguish subsystem activity for each model after the normalization arised from the assumption of the total flux value for each model should be equal in order to allow comparison.

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/fva_subsystem_spans.png}
  \caption[Mean carried fluxes through the ubsystems are shown as stacked bars for strain-wise comparison. Sum of fluxes are normalized to a hundred percent to be able to distinguish flux distribution]{Mean carried fluxes through the ubsystems are shown as stacked bars for strain-wise comparison. Sum of fluxes are normalized to a hundred percent to be able to distinguish flux distribution.}
  \label{fig:fva_subsystem_spans}
  \end{center}
\end{figure}

Despite the similarities in the flux values some of the subsystems, interesting results are observed. The most distinguished models are observed as caffeine, coniferylaldehyde and phenylethanol strains as they do not carry fluxes through fatty acid metabolism reactions. Instead, the caffeine model shows higher activity in the TCA cycle reactions, compared to coniferylaldehyde and phenylethanol strains where the activity increases in the 2-oxocarboxylic acid metabolism.

\section{Survivability Analysis: Knock-out Simulations}
To calculate the essentiality of the reactions and the proteins on the growth, \emph{in-silico} single deletion analyses are done on all the reactions for each model. Growth rate ratios between deletion applied models and the prior models are collected as percentages for each strain. The reactions that have the same value for each experiment are discarded from the results considering they do not differ after evolution. To be able to catch the most important proteins, only the reactions with the minimum of \%95 growth rate decreases are reported here. The impacts of the deleted reactions are seperately shown as heatmaps in Figure \ref{fig:deletion_survivability_rxns} for metabolic reactions and in Figure \ref{fig:deletion_survivability_prots} proteins.

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/deletion_survivability_rxns.png}
  \caption[Heatmaps of the growth rate ratios between deletion strains for each experiment. The knocked-out reactions that cause a decrease higher than \%95 are reported.]{Heatmaps of the growth rate ratios between deletion strains for each experiment. The knocked-out reactions that cause a decrease higher than \%95 are reported.}
  \label{fig:deletion_survivability_rxns}
  \end{center}
\end{figure}

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/deletion_survivability_prots.png}
  \caption[Heatmaps of the growth rate ratios between deletion strains for each experiment. The knocked-out proteins that cause a decrease higher than \%95 are reported.]{Heatmaps of the growth rate ratios between deletion strains for each experiment. The knocked-out proteins that cause a decrease higher than \%95 are reported.}
  \label{fig:deletion_survivability_prots}
  \end{center}
\end{figure}

Although all the models were able to grow in any single deletion on the reactions (with the minimum 45\% lower growth rate from the non-deletion models), 2 enzymes (SAM1 and PMT5) are found essential for all models. Interestingly, MET2, UGA1, COX10, UGA2, HEMI4, THI6 and MRI1 enzymes were essential for all models except the caffeine-resistant model. On the other hand, the deletion of the PGK1, PFK2, TPI1 and PFI1 enzymes decreases the caffeine-resistant model's growth ability higher than other models.

\section{Minimization of Metabolic Adjustment}

Minimization of metabolic adjustment (MOMA) analysis is applied to evolved models to find the closest points to wild-type model simulations (in terms of the Euclidean distances). Analysis is done with the objective of maximization of the growth rate and no constraints applied to the models except for the total protein amount availability. Results are collected for the metabolic reactions and the proteins (as enzyme-draw reactions) seperately. The sum of the distances from the wildtype simulations is used as a ranking system to find most distant reactions and proteins. Individual fluxes through reactions can be seen in Figure \ref{fig:moma_cumdist}.


 The most distant reactions in the evolved models from the wild type model found as palmitoyl-CoA hydrolase, followed by the long chain fatty acid CoA ligase, deoxyuridine kinase, methylglyoxal synthase, 5' nucleotidases, glutathione metabolism reactions. On the other hand, the enzymes TDH1 (glyceraldehyde-3-phosphate dehydrogenase isozyme 1), OLI1 (F0 ATP synthase subunit c), GPT2 (glycerol-3-phosphate/dihydroxyacetone phosphate sn-1 acyltransferase), RIB7 (Diamino-hydroxy-phoshoribosyl-amino-pyrimidine deaminase), SCT1 (glycerol 3-phosphate/dihydroxyacetone phosphate sn-1 acyltransferase) and COB (cytochrome b) were the top distant enzymes in terms of enzyme-draw reaction fluxes.

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/moma_cumdist.png}
  \includegraphics[width=1\columnwidth]{figures/moma_cumdist_rxns.png}
  \caption[Enzymes that have MOMA flux values the most distant from the wild-type flux values in nmol/gDWh]{Enzymes that have MOMA flux values the most distant from the wild-type flux values in nmol/gDWh.}
  \label{fig:moma_cumdist}
  \end{center}
\end{figure}

\section{Sampling Results}

The random solutions for each metabolic models are obtained by maximizing  for a random set of three reactions with random weights. Total of 10,000 solutions for each model are generated after the boundaries on growth rate as an objective function is set to minimum of 90\% and maximum of 100\% growth rate of the flux balance analysis results. The solution spaces for these reactions are plotted as histogram plots where the bin counts, N, are normalized so that the sum(N) is 1 for each analysis.

The most divergent reactions excluding diffusion, transport and exchange reactions from FBA results are plotted in Figure \ref{fig:sampling_fba_top12}, the most divergent enzymes from FVA results are plotted in Figure \ref{fig:sampling_fva_top12}, followed by the MOMA hits in Figure \ref{fig:sampling_moma_top12}.

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/sampling_fba_top12.png}
  \caption[Flux distributions of the reactions that diverge the most in flux balance analysis, obtained from the random sampling of solution spaces for each model shown as histogram plots]{Flux distributions of the reactions that diverge the most in flux balance analysis, obtained from the random sampling of solution spaces for each model shown as histogram plots.}
  \label{fig:sampling_fba_top12}
  \end{center}
\end{figure}

Ferrocytochrome-c:oxygen oxidoreductase reaction shows a great divergence across all models despite having the 0 flux on the caffeine resistant model. This 0 flux on the caffeine model explains the higher fluxes through alcohol dehydrogenase reaction, meaning thet the oxidative phosphorylation rates are lower in the caffeine resistant model compared to ethanol producing pathways. Pyruvate decarboxylase reaction shows almost the pattern with the pyruvate kinase reaction indicating a chain reaction. Remember that the although the models are fed with the same amount of carbon sources, small differences in their reaction fluxes comes from the adaptive evolution expression analyses. That reminded, the small flux values on the 6-phosphogluconolactonase reaction biologically may not mean a big difference, however the different behavior of the silver resistant model (having a wider range over other models) could mean the availability of the althernative pathways for the same biological objective.


\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/sampling_fva_top12.png}
  \caption[Flux distributions of the reactions that diverge the most in flux variability analysis, obtained from the random sampling of solution spaces for each model shown as histogram plots]{Flux distributions of the reactions that diverge the most in flux variability analysis, obtained from the random sampling of solution spaces for each model shown as histogram plots.}
  \label{fig:sampling_fva_top12}
  \end{center}
\end{figure}

Enzyme fluxes are numerically hard to distinguish in histogram plots because of the lower magnitude of the enzyme kinetics. The main target enzymes from the flux variability analysis are found as the glutaredoxin paralogs (GRX1 and GRX2) mostly did not show a very different behaviors on the sampled solution spaces. Glyceraldehyde-3-phosphate dehydrogenase isozymes (TDH1, TDH2 and TDH3) on the other hand show the different preferences between models.

In order to investigate enzyme relationships, correlation coefficients for each enzyme is calculated from the solution spaces for all models. Considering only the correlation values higher than R $>$ 0.5, GRX enzymes are found to be positively correlated with the phospholipid hydroperoxide glutathione peroxidase (GPX) enzymes (Figure \ref{fig:sampling_correlation_grx}).

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/sampling_correlation_grx.png}
  \caption[Correlations of GRX1 (top) and GRX2 (bottom) enzymes to other enzymes calculated from the random sampling of solution spaces. Only the name of enzymes with the correlation value c$>$0.8 are shown in the y-axes]{Correlations of GRX1 (top) and GRX2 (bottom) enzymes to other enzymes calculated from the random sampling of solution spaces. Only the name of enzymes with the correlation value c$>$0.3 are shown in the y-axes}
  \label{fig:sampling_correlation_grx}
  \end{center}
\end{figure}

TDH3 enzyme shows positive correlation only in the caffeine resistant model to 4 enzymes: An enzyme responsible for the regulation of telomerase (CDC13), minor isoform of pyruvate decarboxylase (PDC5), subunit of phosphofructokinase (PFK2), and to a protein of unknown function (EMI2). TDH1 and TDH3 enzymes on the other hand show both positive and negative correlations to multiple enzymes (Figure \ref{fig:sampling_correlation_tdh}).

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/sampling_correlation_tdh.png}
  \caption[Correlations of TDH1 (top), TDH3 (middle) and TDH2 (bottom) enzymes to other enzymes calculated from the random sampling of solution spaces. Only the name of enzymes with the correlation value c$>$0.5 are shown in the y-axes]{Correlations of TDH1 (top), TDH3 (middle) and TDH2 (bottom) enzymes to other enzymes calculated from the random sampling of solution spaces. Only the name of enzymes with the correlation value c$>$0.5 are shown in the y-axes.}
  \label{fig:sampling_correlation_tdh}
  \end{center}
\end{figure}

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/sampling_correlation_oli_gpt_rib.png}
  \caption[Correlations of OLI1 (top), GPT2 (middle) and RIB7 (bottom) enzymes to other enzymes calculated from the random sampling of solution spaces. Only the name of enzymes with the correlation value c$>$0.5 are shown in the y-axes]{Correlations of OLI1 (top), GPT2 (middle) and RIB7 (bottom) enzymes to other enzymes calculated from the random sampling of solution spaces. Only the name of enzymes with the correlation value c$>$0.5 are shown in the y-axes.}
  \label{fig:sampling_correlation_oli_gpt_rib}
  \end{center}
\end{figure}

When we further investigate the enzymatic correlations of OLI1 (F0 ATP synthase subunit c), GPT2 (glycerol-3-phosphate/dihydroxyacetone phosphate sn-1 acyltransferase), and RIB7 (Diamino-hydroxy-phoshoribosyl-amino-pyrimidine deaminase) to other enzymes. We see that RIB7 shows similar results across models with the exception of correlations with HEM4 enzyme (uroporphyrinogen III synthase) absent in ethanolb8, caffeine and phenylethanol resistant models. Similar to HEM4 enzyme, COX15 (heme a synthase), ARH1 (mitochondrial oxidoreductase), NMA1 (nicotinic acid mononucleotide adenylyltransferase) and YAH1 (ferredoxin) enzymes must be investigated since they show different correlations across models although they are all involved in the heme biosynthesis pathways. In the case of OLI1, both positive and negative correlations are observed within all models, although the most distinct strain appears as caffeine resistant model by showying less correlations. GPT2 enzyme positively correlates with YJU3 (monoglyceride lipase) and ALE1 (lysophospholipid acyltransferase) enzymes in wildtype, silver and nickel resistant models with lower correlation values in caffeine resistant model.

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/sampling_moma_top12.png}
  \caption[Flux distributions of the reactions that diverge the most in the minimization of metabolic adjustment analysis, obtained from the random sampling of solution spaces for each model shown as histogram plots]{Flux distributions of the reactions that diverge the most in the minimization of metabolic adjustment analysis, obtained from the random sampling of solution spaces for each model shown as histogram plots.}
  \label{fig:sampling_moma_top12}
  \end{center}
\end{figure}

Glyceraldehyde-3-phosphate dehydrogenase reaction ranks once again #1 in a different analysis, MOMA. It can be seen that separation is clear in all models with a wider flux range in the silver resistant model meaning that its flux value differ in multiple solutions. Followed by palmitoyl-CoA hydrolase, long chain fatty acid-CoA ligase and methylglyoxal synthase reactions are only in the Figure because they are only active in the silver resistant model. However since their flux ranges are too close to 0, they cannot be investigated as solid foundings. Pyruvate kinase reaction shows the same pattern as in previous analyses. Interestingly, proton leak reaction (leakage of hydrogen atoms from cytoplasm to mitochondria) carries flux up to 20 mmol/gDWh, indicating that in the phenylethanol, coniferylaldehyde and nickel resistant models, more than available proton is required in the mitochondria. Lastly, 5'-nucleotidase reactions, guanosine kinase and pyridoxamine kinase reactions are only active for silver resistant model, same as the previously mentioned reactions.

Finally, mean values for each enzyme from solution spaces are calculated. Z-scores are obtained from the difference between this means in each strain model to the wildtype model fluxes, divided by
the square root of the sum of standard deviations of this difference. Z-scores for enzymes are converted to p-values within gaussian distribution, and compared to the p-values obtained from the differential expression analysis. Discarding the non-significant enzymes in both cases, the following Figure \ref{fig:sampling_regulation_heatmaps} is constructed to compare regulations.

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/sampling_regulation_heatmaps.png}
  \caption[Regulation of the enzymes obtained from the random sampling of solution spaces (F) and differential expression analysis (E) in all models. Enzymes that only have significant (p$<$0.05) changes are plotted in both flux and expression changes]{Regulation of the enzymes obtained from the random sampling of solution spaces (F) and differential expression analysis (E) in all models. Enzymes that only have significant (p$<$0.05) changes are plotted in both flux and expression changes. }
  \label{fig:sampling_regulation_heatmaps}
  \end{center}
\end{figure}
