\chapter{RESULTS}

\section{Model Constraints and Curation}


To make sure the \emph{in-silico} growth rate predictions are in agreement with the physiological kinetic parameters obtained from the laboratory experiments, fine adjustment on the chosen metabolic model is a requirement. Since the growth-associated maintenance (GAM) and non-growth associated maintenance (NGAM) reactions in a model play a determinant role in the simulation results, fluxes through these reactions must be constrained to a fixed value. Flux of the NGAM reaction is constrained to 0.7 mmol/gDWh\textsuperscript{-1} for aerobic model, and 0 mmol/gDWh\textsuperscript{-1} for anaerobic model simulations as calculated in the previous studies \cite{nilsson2016metabolic}. Since the GAM depends on the biomass composition defined, results of a chemostat experiment \cite{van1998effect} are used as a guide to fit simulation predictions to. The model is simulated iteratively with a range of values for the GAM and the best fit is found at the level of 31.4 mmol/gDWh\textsuperscript{-1} (Figure \ref{fig:gam_fitting}).

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/gamfitting.png}
  \caption[The required flux for the growth associated maintenance reaction is determined by the fitting simulation results to the experimental data]{The required flux for the growth associated maintenance reaction is determined by the fitting simulation results to the experimental data. The best fit is found at 31.4 mmol/gDWh\textsuperscript{-1} and the reaction is constrained to that flux value.}
  \end{center}
  \label{fig:gam_fitting}
\end{figure}

Average enzyme saturation value is set to 50\% where the lowest error rate is obtained as a fraction to biomass (Figure \ref{fig:sigma_fitting}). By constraining the total enzyme mass, the metabolic model became able to show overflow metabolism without any other constaint applied.

\begin{figure}[H]
  \begin{center}
  \includegraphics[width=1\columnwidth]{figures/sigmafitting.png}
  \caption[Average enzyme saturation factor for the GECKO method is determined by the iterative simulations for the lowest absolute relative error to the experimantal data]{Average enzyme saturation factor for the GECKO method is determined by the iterative simulations for the lowest absolute relative error to the experimantal data. The value is set to 50\% where the lowest error rate is obtained.}
  \end{center}
  \label{fig:sigma_fitting}
\end{figure}

To prevent over-constraining the model with the enzyme kinetics, the most limiting proteins in the model are found based on their sensitivity on the objective function, i.e. growth. The $k_{cat}$ values for the the limiting enzymes are updated with the maximum available values from BRENDA\cite{jeske2019brenda} database by automated iterations until the simulation results for the growth rate agrees with the experimental growth rate. Modifications to the $k_{cat}$ values with their objective control coefficients can be found in the Table \ref{table:gecko_iterations}.

\input{tables/gecko_iterations.tex}

\vspace{-0.5cm}
\section{Differential Expression Analysis and Integration}

The obtained expression datasets were quantile-normalized for their own experiments. Three replicates for both reference and the evolved strains for each experiment (exception with the ethanol-b8 strain which has two replicates) is used for the differential expression analysis. The distribution across normalized gene expression levels across each experiment can be seen in the Figure \ref{fig:expr_boxplot}. No further normalization applied to prevent introducing errors and datasets are used as obtained from the database.

\begin{figure}[H]
\begin{center}
\includegraphics[width=1\columnwidth]{figures/expr_boxplots.png}
\caption[Boxplots of the normalized gene expression levels]{Boxplots of the normalized gene expression levels obtained from Gene Expression Omnibus. Quantile normalization performance is indicated by the medians (black horizontal lines) that are almost at the same level with their reference reads.}
\end{center}
\label{fig:expr_boxplot}
\end{figure}

A statistical threshold (p $<$ 0.05) for the differential analysis of the normalized gene expression levels detected a total of 1606 genes in ethanol-b2, 2947 genes in ethanol-b8, 4743 genes in caffeine, 2267 genes in coniferylaldehyde, 3448 genes in iron, 152 genes in nickel, 4796 genes in phenylethanol, and 4796 genes in silver resistant strains differentially expressed. Commonly upregulated and downregulated genes frequencies across all strains can be found in the Figure \ref{fig:intersections_by_degree}.



\begin{figure}[H]
\begin{center}
\includegraphics[width=1\columnwidth]{figures/intersections_by_degree.png}
\caption[UpSet plot showing the number of common genes that are differentially expressed significantly]{UpSet plot showing the number of common genes that are differentially expressed significantly (p $<$ 0.05) across evolved strains. Connected black circles in the below panel's matrix represents a certain intersection between experiment strains, and the number of genes in that intersection are shown in the top bar graphs. Differentially expressed genes are seperately plotted for (A) upregulated genes, and (B) downregulated genes.}
\end{center}
\label{fig:intersections_by_degree}
\end{figure}

Despite the high number of differentially expressed genes in the silver resistant strain, the intersection with the most number of strains discluded it. There were not any common differentially expresssed genes considering all strains. The most common genes (as shown in the first bars in the UpSet plot) are reported in Table \ref{table:common_genes}.

\begin{table}[H]
\small
\vskip\baselineskip
  \begin{center}
  \caption[Common upregulated and downregulated genes]{The common upregulated and downregulated genes within ethanolb2, ethanolb8, caffeine, coniferylaldehyde, iron, nickel and phenylethanol resistant strains.}
  \begin{tabular}{|p{4cm}|p{10cm}|}
     \hline
    \textbf{Upregulated Genes} & \textbf{Downregulated Genes}  \\
      \hline
      YDR075W, YDR492W, YGR239C, YIL118W, YLR215C, YML005W, YML125C, YNL024C, YNL188W, YNR044W, YOL002C, YOR107W, YPR052C & YBL015W, YBR132C, YBR285W, YBR299W, YCR068W, YDL238C, YDR055W, YER033C, YGL146C, YGL250W, YGR023W, YGR149W, YGR161C, YGR288W, YGR289C, YGR292W, YIL097W, YIR017C, YJL042W, YJL053W, YJL082W, YJL132W, YLR092W, YLR176C, YLR240W, YLR446W, YMR052C-A, YMR053C, YMR081C, YMR103C, YMR104C, YMR105C, YMR160W, YMR194C-A, YMR258C, YMR291W, YNL093W, YNL277W, YNR001C, YOL117W, YOR027W, YOR132W, YOR178C, YOR230W, YPR079W, YPR154W \\ \hline
  \end{tabular}

  \label{table:common_genes}
  \end{center}
\end{table}



The log2(fold-change) values of differentially expressed genes and their frequencies are plotted for each experiment in Figure \ref{fig:expr_frequencies_before}. Although the high number of differentially expressed genes are collected, due to the limited reaction number in the genome-scale metabolic models, only a part of obtained fold-change expression values were able to integrated into the models. Because of the limitations in the model, no further cutoff considered for the fold change values.

\begin{figure}[H]
\begin{center}
\includegraphics[width=1\columnwidth]{figures/expr_frequencies_before.png}
\caption[Frequencies of the differentially expressed genes after the gene expression analysis]{Frequencies of the differentially expressed genes after the gene expression analysis}
\end{center}
\label{fig:expr_frequencies_before}
\end{figure}

Total of 277 genes in ethanol-b2, 464 genes in ethanol-b8, 783 genes in caffeine, 374 genes in coniferylaldehyde, 552 genes in iron, 28 genes in nickel, 759 genes in phenylethanol and 759 genes in silver resistant strains were integrated into the metabolic model to generate strain-specific models (\ref{fig:expr_frequencies_after}).

\begin{figure}[H]
\begin{center}
\includegraphics[width=1\columnwidth]{figures/expr_frequencies_after.png}
\caption[Frequencies of the differentially expressed genes that are integrated into the metabolic model]{Frequencies of the differentially expressed genes that are integrated into the metabolic model.}
\end{center}
\label{fig:expr_frequencies_after}
\end{figure}

In order to validate the expression data integration method, fluxes for each enzyme (i.e. protein flux from protein pool to reaction for each protein) is predicted by flux balance analysis on all the strain models and on the wild-type model (wild-type here is referred as the metabolic model without expression data integrated). Flux ratios between wild-type and evolved models are calculated, and linear regression analysis is fitted after elimination of the outliers by the generalized extreme studentized deviate test to compare the flux-changes with the protein fold-changes from the expression data. Results confirm a good correlation between simulations and protein fold-change values (Figure \ref{fig:expr_linear_regressions}).

\begin{figure}[H]
\begin{center}
\includegraphics[width=0.8\columnwidth]{figures/expr_linear_regressions.png}
\caption[Linear regression analyses of the fold-changes]{Linear regression analyses of the differential expression fold-changes obtained from microarray studies and the enzyme-draw flux fold-changes obtained from flux balance analysis simulations of corresponding proteins.}
\end{center}
\label{fig:expr_linear_regressions}
\end{figure}



\section{Flux Balance Analysis Simulations}

The metabolic model is first simulated without the integration of the expression data (this model will be referred as wild-type model). Since the model is already enzymatically constrained, the only additional constraint required is the total amout of enzymes available in the enzyme pool. All the lower bounds for reactions were set to 0, and the upper bounds were set to 1000, except for the enzyme pool reaction. By doing that, the model was set free to be able to allocate enzymes as the reactions require. Flux rates of several exchange reactions as a function of growth rate are plotted in Figure \ref{fig:wt_crabtree}.

A behavioral change is observed around the time when growth rate is approximately 0.3 h\textsuperscript{-1}. Since the model is simulated under fully areobic conditions (no constraints applied) and produces ethanol, this behavior can be explained by the Crabtree effect, where the yeast metabolism switches to perform both respiration and fermantation at the same time at a critical specific growth rate.

\begin{figure}[H]
\begin{center}
\includegraphics[width=1\columnwidth]{figures/wt_crabtree.png}
\caption[Metabolic model shows the overflow metabolism]{Metabolic model shows the overflow metabolism (crabtree effect) under fully aerobic conditions.}
\end{center}
\label{fig:wt_crabtree}
\end{figure}
\vspace{-1.0cm}

Robustness analysis is performed on the wild-type model for the growth rate with the varying levels of glucose uptake, oxygen uptake, acetate secretion and ethanol secretion rates (Figure \ref{fig:wt_robustness}). Unlike the traditional metabolic models, enzymatically constrained model showed non-linear graphs.

\begin{figure}[H]
\begin{center}
\includegraphics[width=1\columnwidth]{figures/wt_robustness.png}
\caption[Robustness analyses on the glucose uptake, oxygen uptake, acetate secretion and ethanol secretion rates]{Robustness analyses on the glucose uptake, oxygen uptake, acetate secretion and ethanol secretion rates.}
\end{center}
\label{fig:wt_robustness}
\end{figure}

%For the anaerobic conditions, double robustness analysis of oxygen uptake and ethanol secretion rate is perfomed on the model to show the effect of oxygen availability on the system (Figure \ref{fig:v2_wt_robustness_ethanol}). The model predicted higher ethanol secretion under the lower oxygen availibility, as expected.

%\begin{figure}[H]
%\begin{center}
%\includegraphics[width=0.8\columnwidth]{v2_wt_robustness_ethanol.png}
%\caption[Double robustness analysis on oxygen uptake and ethanol secretion reactions]{Double robustness analysis on oxygen uptake and ethanol secretion reactions. (A) Phenotype phase-plane, (B) shadow prices of reactions.}
%\end{center}
%\label{fig:v2_wt_robustness_ethanol}
%\end{figure}

After the validation of the wild-type model, all models for strains are simulated for the growth rates as a function of iteratively increasing glucose uptake rates (Figure \ref{fig:growth_glucose_ales}). Models were able to grow up to the point where the protein accesibility becomes a limiting factor. It has been found that the evolution models except for the caffeine tolerant model show similar patterns on the points where the enzymes become limiting (break points in the lines). Caffeine tolerant model shows different pattern in a way that it can consume more glucose to reach higher growth rates compared to others. Also it must be noted that while only caffeine and coniferylaldehyde tolerant models can grow at higher rates than wild-type model, ethanol, iron, nickel, phenylethanol and silver models can grow at lower maximum rates. The maximum growth rates and corresponding glucose uptake rates for each model are summarized in the Table \ref{table:growth_glucose_table}.

\begin{figure}[H]
\begin{center}
\includegraphics[width=1\columnwidth]{figures/growth_glucose_ales.png}
\caption[Growth rates as a function of iteratively increasing glucose uptake rates]{Growth rates as a function of iteratively increasing glucose uptake rates.}
\end{center}
\label{fig:growth_glucose_ales}
\end{figure}
\vspace{-1.0cm}



\begin{table}[H]
\small
\vskip\baselineskip
  \begin{center}
  \caption[Maximum growth rates obtained from flux balance analysis simulations for each strain and their maximum glucose uptake rates (GUR) required for the growth]{Maximum growth rates obtained from flux balance analysis simulations for each strain and their maximum glucose uptake rates (GUR) required for the growth.}
    \vspace{5mm}
  \begin{tabular}{|c|c|c|}
     \hline
    \textbf{Experiment} & \textbf{Growth Rate (1/h)} & \textbf{GUR (mmol/gDWh)} \\
      \hline
      ethanolb2           & 0.35466                    & 14                                       \\ \hline
      ethanolb8           & 0.34736                    & 16                                       \\ \hline
      caffeine            & 0.46769                    & 28                                       \\ \hline
      coniferylaldehyde   & 0.39132                    & 20                                       \\ \hline
      iron                & 0.37554                    & 15                                       \\ \hline
      nickel              & 0.37523                    & 18                                       \\ \hline
      phenylethanol       & 0.34382                    & 18                                       \\ \hline
      silver              & 0.34163                    & 25                                       \\ \hline
      wildtype            & 0.37618                    & 19                                       \\ \hline
  \end{tabular}

  \label{table:growth_glucose_table}
  \end{center}
\end{table}

In order to characterize the differentiating reactions in the flux balance analysis solution vectors, cumulative flux vectors of each experiment are plotted (Figure \ref{fig:cumflux_free}). Standart deviations for each reaction between strains are calculated to find most diverging points and plotted as a bar graph.

\begin{figure}[H]
\begin{center}
\includegraphics[width=1\columnwidth]{figures/cumflux_free.png}
\caption[Cumulative flux vectors of each simulation and standart deviations]{Cumulative flux vectors of each model and the standart deviations of reactions across each adaptatoin.}
\end{center}
\label{fig:cumflux_free}
\end{figure}

Reactions with the standart deviation values more than 10 mmol/gDWh are listed in Table \ref{table:cumulative_free_stds}. The same reactions with different numbers in their names use different enzymes. For example, glyceraldehyde-3-phosphate dehydrogenase (GADPH) reaction can be carried out in the presence of YGR192C or YJL052W or YJR009C genes' proteins, in other words, TDH1 or TDH2 or TDH3 isozymes. Therefore there are No1, No2 and No3 versions of that reaction. It must be noted that these multiple reactions carried out with isozymes in the metabolic model may not mean a significant difference if the enzymes do not catalyze any other reaction. This means that the chosen isozyme does not matter if the flux value across isozymes are the same.
\input{tables/cumulative_free_stds.tex}

Cumulative flux vectors of each experiment are plotted once more when the boundaries of the glucose uptake reaction is constrained to 10 gDWh\textsuperscript{-1} for each model (Figure \ref{fig:cumflux_gur10}). This has done to eliminate differences might be arising because of the free glucose uptake rates (as shown in below section, each model can uptake different maximum amount of glucose) and therefore affecting the downstream reaction fluxes.

\begin{figure}[H]
\begin{center}
\includegraphics[width=1\columnwidth]{figures/cumflux_gur10.png}
\caption[Cumulative flux vectors of each experiment when glucose uptake rate is constrained]{Cumulative flux vectors of each simulation and standart deviations. Glucose uptake rate is constrainted to  10 mmol/gDWh for all models. }
\end{center}
\label{fig:cumflux_gur10}
\end{figure}

Reactions with the standart deviation values more than 5 mmol/gDWh are listed in Table \ref{table:cumulative_free_stds}. Despite the decrease in the flux values (caused by the lower uptake rates of glucose), the top divergent reactions obtained are in agreement with the no-constraint flux balance analysis results.

\input{tables/cumulative_gur10_stds.tex}

Finally, only arm reactions (where the isozyme reactions are combined, see Methods) when the glucose uptake rate is constrained to 10 mmol/gDWh are plotted as heatmap in Figure \ref{fig:flux_stds_heatmap} after normalization. As it can be seen from the previous tables, most of the differences are caused by the isozymes with the flux values close to each other. By considering only arm reactions and a fixed glucose uptake rate, the reactions that are actually differ because of the expression integration are found.



\begin{figure}[H]
\begin{center}
\includegraphics[width=1\columnwidth]{figures/flux_stds_heatmap.png}
\caption[Heatmap plot of the reaction fluxes where the standart deviation between models are highest. Reaction names with the reversible tag means intake of the substrate, while the arm tag means that the flux of that particular reaction is a sum of multiple isozyme reactions]{Heatmap plot of the reaction fluxes where the standart deviation between models are highest. Reaction names with the reversible tag means intake of the substrate, while the arm tag means that the flux of that particular reaction is a sum of multiple isozyme reactions. }
\end{center}
\label{fig:flux_stds_heatmap}
\end{figure}


\section{Flux Variability Analysis}
Flux ranges of each enzyme on models are analyzed by the linear problem in the Flux Variability Analysis method where the objective functions to minimize and maximize all reactions iteratively with growth as the objective function kept at 90\%. Minimum and maximum available fluxes are collected in the iterative process for each reaction, and results are plotted in Figure \ref{fig:v2_fva}). Standart deviations on flux ranges for each reaction are also plotted to find most divergent targets. The most divergent proteins (top 20) across all experiments are collected in Table \ref{table:fva_results}.

\begin{figure}[H]
\begin{center}
\includegraphics[width=1\columnwidth]{v2_fva.png}
\caption[Flux variability analysis]{Flux variability analysis results sorted by the wild-type flux ranges.}
\end{center}
\label{fig:v2_fva}
\end{figure}
\input{tables/fva_results.tex}


\section{Enzyme Essentiality}
To calculate the essentiality of the proteins, \emph{in-silico} single reaction deletion analysis is applied to the protein draw reactions on all models. The impacts of the enzyme deletions on growth (as the objective function) of the most essential (top 10) proteins are plotted in Figure \ref{fig:v2_proteinknockout}.

\begin{figure}[H]
\begin{center}
\includegraphics[width=1\columnwidth]{v2_proteinknockout.png}
\end{center}
\caption[Important protein targets and their knock-out effects on growth rate]{Important protein targets and their knock-out effects on growth rate.}
\label{fig:v2_proteinknockout}
\end{figure}


\section{Minimization of Metabolic Adjustment}

MOMA results are investigated for the metabolic reactions and the protein draw reactions seperately. Flux distances for each reaction of evolved models to the wild type models are calculated and plotted (Figure \ref{fig:v2_moma_cumdist}). The top 5 ranked reactions for the maximum cumulative distance from the wild type fluxes found as palmitoyl-CoA hydrolase, long chain fatty acid CoA ligase, carbon dioxide exchange, glyceraldehyde-3-phosphate dehydrogenase, methylglyoxal synthase reactions. On the other hand, the enzymes TDH1 (glyceraldehyde-3-phosphate dehydrogenase isozyme 1), RIB7 (Diamino-hydroxy-phoshoribosyl-amino-pyrimidine deaminase), GPT2 (glycerol-3-phosphate/dihydroxyacetone phosphate sn-1 acyltransferase), SCT1(glycerol 3-phosphate/dihydroxyacetone phosphate sn-1 acyltransferase) and SEC53 (phosphomannomutase) were the top 5 ranked enzymes.

\begin{figure}[H]
\begin{center}
\includegraphics[width=1\columnwidth]{v2_moma_cumdist.png}
\noindent\rule{14cm}{0.4pt}
\vskip\baselineskip
\includegraphics[width=1\columnwidth]{v2_moma_cumdist_prot.png}
\end{center}
\caption[Reactions and proteins that has MOMA fluxes the most distant from the wild-type fluxes]{Reactions (top) and proteins (bottom) that have MOMA flux values the most distant from the wild-type flux values.}
\label{fig:v2_moma_cumdist}
\end{figure}


\section{Sampling Results}
I was not able to sample solution spaces correctly, yet.
